WordPress/src/main/java/org/wordpress/android/datasets/PeopleTable.java

```diff
@@ -26,6 +25,7 @@ private static SQLiteDatabase getWritableDb() {
     public static void createTables(SQLiteDatabase db) {
         db.execSQL("CREATE TABLE " + PEOPLE_TABLE + " ("
                 + "person_id               INTEGER DEFAULT 0,"
+                + "site_id                 TEXT,"
```

Comment:
BTW, notice that I didn't propose `local_site_id` but just `blog_id`. At the end, the table will have both `blog_id` and `local_blog_id` which I think makes it clear that one is local the other is not.

I don't particularly mind moving to using "site_id" instead though. It will probably make it more aligned with the API documentation as well, and that's a good thing.

Not sure what you mean about the `SiteSettings` though... not seeing a "site_id" or similar to its table.